lib/_http_common.js

```diff
@@ -233,70 +233,9 @@ function httpSocketSetup(socket) {
   socket.on('drain', ondrain);
 }
 
-/**
- * Verifies that the given val is a valid HTTP token
- * per the rules defined in RFC 7230
- * See https://tools.ietf.org/html/rfc7230#section-3.2.6
```

Comment:
I would like to keep this comment. When looking through the code it is good to have a reference handy.

Reply:
@BridgeAR Restored.